content/en/docs/tasks/security/authorization/authz-jwt/index.md

```diff
@@ -0,0 +1,182 @@
+    200
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output, retry after a few seconds.
+Caching and propagation can cause a delay.
+{{< /warning >}}
+
+## Allow requests with valid JWT and list-typed claims
+
+1. The following command creates the `jwt-example` request authentication policy
+for the `httpbin` workload in the `foo` namespace. The policy allows the Envoy proxy in front of
+the `httpbin` workload to accept a JWT issued by `testing@secure.istio.io`.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "RequestAuthentication"
+    metadata:
+      name: "jwt-example"
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      jwtRules:
+      - issuer: "testing@secure.istio.io"
+        jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+    EOF
+    {{< /text >}}
+
+1. Verify that a request with an invalid JWT is denied.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -s -o /dev/null -H "Authorization: Bearer invalidToken" -w "%{http_code}\n"
+    401
+    {{< /text >}}
+
+1. Verify that a request without a JWT is allowed because there is no authorization policy.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+1. The following command creates the `require-jwt` authorization policy for the `httpbin` workload in the `foo` namespace.
+The policy requires all requests to the `httpbin` workload to have a valid JWT with
+`requestPrincipal` set to `testing@secure.istio.io/testing@secure.istio.io`.
+Istio constructs the `requestPrincipal` by combining the `iss` and `sub` of the JWT token
+with a `/` separator as shown.
+
+    {{< text syntax="bash" expandlinks="false" >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: require-jwt
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           requestPrincipals: ["testing@secure.istio.io/testing@secure.istio.io"]
+    EOF
+    {{< /text >}}
+
+1. Get the JWT that sets the `iss` and `sub` keys to the same value, `testing@secure.istio.io`.
+This causes Istio to generate the attribute `requestPrincipal` with the value,
+`testing@secure.istio.io/testing@secure.istio.io`.
+
+    {{< text syntax="bash" expandlinks="false" >}}
+    $ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/demo.jwt -s) && echo $TOKEN | cut -d '.' -f2 - | base64 -d -
```

Comment:
Nice work sorting this out !

`{{< github_file >}}` usage is described here: https://istio.io/about/contribute/creating-and-editing-pages/#github

TLDR: use `@` when using a bash quote. use `{{< github_file>}}` to create a link to a file. If as in this example line 113 contains `@` that is not an expansion, use `expandlinks="false"`. This will create a problem in the infrastructure mentioned above.

Reply:
(no action needed).